last of 1.55.0 doc (#3464)
FusionAuth/fusionauth-site: astro/src/content/docs/apis/_application-request-body.mdx

    * `SlidingWindow` - the expiration is calculated from the last time the token was used.
    * `SlidingWindowWithMaximumLifetime` - the expiration is calculated from the last time the token was used, or until the <InlineField>maximumTimeToLiveInMinutes</InlineField> is reached. <span class="text-green-600">Available since 1.46.0</span>
  </APIField>

>   <APIField name="application.jwtConfiguration.refreshTokenOneTimeUseConfiguration.gracePeriodInSeconds" type="Integer" optional defaults="0" since="1.55.0">

Reviewer: ```suggestion
  <APIField name="application.jwtConfiguration.refreshTokenOneTimeUseConfiguration.gracePeriodInSeconds" type="Integer" optional defaults="0" since="1.55.1">
```

>     The length of time specified in seconds that a one time use token can be re-used.

Reviewer: ```suggestion
    The length of time specified in seconds that a one-time use token can be re-used.
```

>     This value must be greater than `0` and less than `86,400` which is equal to 24 hours. Setting this value to `0` effectively disables the grace period which means a one-time token may not be reused. For security reasons, you should keep this value as small as possible, and only increase past `0` to improve reliability for an asynchronous or clustered integration that may require a brief grace period.

Reviewer: Should we specify the max without a comma, since putting the comma-formatted value generates an error when trying to submit the form?

Author: We have plenty of examples in doc where we format the number with a comma. I am ok with the comma personally, I think it make sit more readable. 

I can do some more searching to see if we have precedence for not using a comma.

>     Note that one-time use tokens refreshed within a grace period are not considered for revocation when <InlineField>tenant.jwtConfiguration.refreshTokenRevocationPolicy.onOneTimeTokenReuse</InlineField> is `true`. When a token is re-used within the grace period the current token will be returned on the API response and the token will not be rotated.

Reviewer: ```suggestion
    Note that one-time use tokens refreshed within a grace period are not considered for revocation when <InlineField>tenant.jwtConfiguration.refreshTokenRevocationPolicy.onOneTimeTokenReuse</InlineField> is `true`. When a token is reused within the grace period the current token will be returned on the API response and the token will not be rotated.
```
